src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -1269,9 +1278,10 @@
 
 (defn- completion-candidates-for-current-ns []
   (let [cur-ns @current-ns]
-    (into (completion-candidates-for-ns cur-ns true)
-      (comp (mapcat keys) (map str))
-      ((juxt :renames :rename-macros :uses :use-macros) (get-namespace cur-ns)))))
+    (concat (completion-candidates-for-spec cur-ns)
+            (into (completion-candidates-for-ns cur-ns true)
```

Comment:
I am not entirely sure we need this. It gets spec completions for the current namespace. What do you think?

Reply:
might be needed for `::a-spec`-style completions?